frontend/packages/shared/src/components/TreeView/TreeViewItem/TreeViewItem.module.css

```diff
@@ -13,7 +13,15 @@
   border-width: 0 0 0 var(--vertical-line-width);
   color: var(--fds-semantic-text-neutral-default);
   justify-content: left;
+}
+
+.button[aria-selected='true']:not(:hover) {
   width: 100%;
+  pointer-events: none;
+}
+
+.button[aria-selected='false']::after {
+  z-index: -1;
 }
```

Comment:
Jeg testet dette lokalt, og det fungerer litt av og på. Det største problemet er kanskje at det ikke lengre fungerer så bra i andre nettlesere. Her må vi nok grave mer. Det er mulig at `pointer-events` er rett spor, men den blokkerer muligheten til å velge et element ved å klikke på knappen. Det skurrer også litt at den settes på `:not(:hover)`.

Reply:
Jeg har testet mange ganger i Firefox and edge, og de funket bra, i hvilket nettleseren de funker ikke helt?

Reply:
Testet i Firefox og Chrome. I begge var det ganske ustabilt.